### PR TITLE
[POC] ES search after

### PR DESCRIPTION
**Pull request by jjanvier**
Where is the PQB used?
* list products in the API
* command to query products => cursor
* command to remove products => cursor
* group saver => cursor
* variant group updater => cursor
* group updater => cursor
* product repository => cursor
* database product reader (export) (+mass edit for variant group) => cursor
* product datasource + product association datasource (grid) => must dig this one
* EE - rule selector => cursor
* EE - mass publish products => cursor
* EE - teamwork assistant product repository => cursor

[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Datasource/ProductDatasource.php [hunk 1]
```diff
@@ -28,11 +32,13 @@ class ProductDatasource extends Datasource
     public function __construct(
         ObjectManager $om,
         HydratorInterface $hydrator,
-        ProductQueryBuilderFactoryInterface $factory
+        ProductQueryBuilderFactoryInterface $factory,
+        NormalizerInterface $normalizer
```
Comment (anaelChardan) [inline src/Pim/Bundle/DataGridBundle/Datasource/ProductDatasource.php:36]: = null to avoid bcs ? :)

Comment (jjanvier) [reply src/Pim/Bundle/DataGridBundle/Datasource/ProductDatasource.php:36]: Just a dirty POC

Don't comment this PR ;)

We'll do another one :)

Comment (anaelChardan) [reply src/Pim/Bundle/DataGridBundle/Datasource/ProductDatasource.php:36]: Ok, sorry :D

**Comment by jjanvier** [comment]
has been resolved by https://github.com/jjanvier/pim-community-dev/pull/97 and https://github.com/jjanvier/pim-community-dev/pull/108

**Pull request closed without merging**